product_template_multi_link/tests/test_product_template_link_type.py

```diff
@@ -0,0 +1,262 @@
+# Copyright 2019 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from psycopg2 import IntegrityError
+
+from odoo.tests import TransactionCase
+from odoo.tools import mute_logger
+
+
+class TestProductTemplateLinkType(TransactionCase):
```

Comment:
You can inherit from `BaseCommon` class and remove the `tracking_disable=True` key from context set in `setUpClass` method